src/state/login/index.js

```diff
@@ -1,7 +1,8 @@
 import { handleActions, createAction } from 'redux-actions';
 
 const initialState = ({
-  loading: false
+  loading: false,
+  invalidLogin: 'false'
```

Comment:
why string?

Reply:
You haven't seen that.. i was checking sth while doing test.. nevermind :D